version/VERSION

```diff
@@ -0,0 +1 @@
+0.31.0-dev
```

Comment:
Is the expectation that we'll set the version back to `-dev` in the first commit after successful release?

Reply:
I've seen teams doing a [stage commit](https://github.com/hashicorp/consul/commit/bd257019c580a7f06b3ef99168698929b74f4bfc) to get everything ready for a release and then re-add the `-dev` prerelease string afterwards.